src/crepe/system/InputSystem.h

```diff
@@ -31,12 +31,13 @@ class InputSystem : public System {
 
 private:
 	//! Stores the last position of the mouse when the button was pressed.
-	std::pair<int, int> last_mouse_down_position{INFINITY, INFINITY};
+	ivec2 last_mouse_down_position{std::numeric_limits<int>::max(),
+								   std::numeric_limits<int>::max()};
```

Comment:
Moet dit echt specifiek `std::numeric_limits<int>::max()` zijn? Waarom niet gewoon `{0, 0}` of helemaal niks omdat `ivec2` RAII is?

Reply:
ik dacht dat er volgens de code standaard altijd een standaard waarde was en 0,0 kan in de game units voorkomen maar het zal eigenlijk nooit voor problemen zorgen

Reply:
In de code standaard staat
> C++-style structs should define default values for all <b><ins>non-trivial</ins></b> fields.
en `ivec2` is geen triviaal type omdat deze zijn eigen members initialiseert met de default constructor